fc-community-backend/apps/api/src/factchain-core/nftService.ts

```diff
@@ -0,0 +1,134 @@
+import axios from "axios";
+import FormData from "form-data";
+import Replicate from "replicate";
+import { Note } from "./types";
+
+export class NFTService {
+
+  static generateNoteImage = async (note: Note, count = 0): Promise<string> => {
+    if (note.content === undefined) {
+      throw new Error("Can't generate image from empty note!");
+    }
+    const replicate = new Replicate({
+      auth: process.env.REPLICATE_API_TOKEN,
+    });
+
+    try {
+      const replicateUrlObj = await replicate.run(
+        "stability-ai/sdxl:c221b2b8ef527988fb59bf24a8b97c4561f1c671f73bd389f866bfb27c061316",
+        {
+          input: {
+            width: 768,
+            height: 768,
+            prompt: note.content,
+            refine: "expert_ensemble_refiner",
+            scheduler: "K_EULER",
+            lora_scale: 0.6,
+            num_outputs: 1,
+            guidance_scale: 7.5,
+            apply_watermark: false,
+            high_noise_frac: 0.8,
+            negative_prompt: "",
+            prompt_strength: 0.8,
+            num_inference_steps: 25,
+          },
```

Comment:
Can you add some comments on the why of some of these values? Just so we know which ones are required to work, and which ones are trial and error that we can play with.

Also what's the magic first arg?

Reply:
Commented, the magic first arg is the replicate AI model Identifier to run